main.go

```diff
@@ -103,7 +106,24 @@ func main() {
 	if *webhook {
 		startWebhookServer(c, *disableWebhookConfigInstaller, *webhookPort)
 	} else if *dashboard {
-		startDashboardServer(c, *auditPath, *dashboardPort, *dashboardBasePath)
+		if *loadAuditFile == "" {
```

Comment:
I'm down to have a quick working session if you want to go back and forth on some ideas of where this logic should live. I am currently thinking that this should all be in the dashboard pkg and you should always pass `loadAuditFile` to `startDashboardServer()`. 

I'm happy to pair up and talk through your reasoning for this so I can better understand the problem.

Reply:
Even though this functionality is currently only used by the dashboard, loading an audit from a file doesn't really seem like a dashboardy thing to do.

I'd suggest putting the loadAuditDataFromFile fn in the same package where the AuditData type is defined, i.e. `pkg/validator`

Reply:
I think we're saying the same thing but I haven't been super clear. I mean the actual marshalling of the file logic should definitely be in the validator pkg but calling the `LoadAuditFromFile()` should happen in dashboard instead of main. If we're not saying the same thing I'm happy to connect in slack :)

Reply:
Ah no, that SGTM